package.json

```diff
@@ -87,47 +81,35 @@
   "browserslist": [
     "last 2 Chrome versions",
     "last 2 iOS major versions",
-    "ie 11",
     "last 2 Firefox versions",
     "last 2 Edge versions",
     "last 2 Safari major versions"
   ],
   "scripts": {
-    "build": "npm run ciBuild",
-    "apiBuild": "wca analyze 'src/auro-datetime.js' --outFiles docs/api.md",
+    "build": "npm-run-all build:sass sass:render cssLint dist:js bundler postinstall build:api",
```

Comment:
There are a lot of legacy dependencies in here that should be removed. Please review the generator template.

Reply:
Done.